http-server/src/test/java/org/triplea/server/lobby/CacheUtilsTest.java

```diff
@@ -83,12 +79,13 @@ void refreshTrueWhenFoundInCache() {
 
     @Test
     void refreshedItemsAreWrittenBackIntoTheCache() {
-      when(mockCache.getIfPresent(KEY)).thenReturn(VALUE);
+      final Cache<String, Integer> cache = CacheBuilder.newBuilder().build();
+      cache.put(KEY, VALUE);
 
-      final boolean result = CacheUtils.refresh(mockCache, KEY);
+      final boolean result = CacheUtils.refresh(cache, KEY);
 
       assertThat(result, is(true));
-      verify(mockCache).put(KEY, VALUE);
```

Comment:
I think we might be losing the purpose of this test. We are checking that a `refresh` will put the value back into the cache again. `refresh` could be implemented as a no-op and this test would still pass. If using a live cache, we could try to test the same thing by waiting for a cache key to almost be expired. I gave that a try as a first implementation when writing this test but found it to be brittle. EG:

```
T0: write key that expires at T10
T5: refresh key, extends expiry to T15
T12: verify key value still exists, verifies that key was refreshed and did not expire at T10 
```

The mock object route version is more deterministic and hence the preference for it in this test.